modules/dag-shared/src/main/scala/org/tessellation/dag/block/BlockValidator.scala

```diff
@@ -25,14 +25,17 @@ import org.tessellation.schema.transaction.{Transaction, TransactionReference}
 import org.tessellation.security.SecurityProvider
 import org.tessellation.security.signature.Signed
 
+import eu.timepit.refined.auto._
```

Comment:
is it required here?

Reply:
We use it throughout the repo so I imported it instead of extracting int with `.value`

Reply:
that's what I asked about :) if it's really used or just imported because we import it everywhere.

Reply:
Yep, used!

Reply:
Also you can put `PosInt` in place where `Int` is required